source/mir/random/engine/package.d

```diff
@@ -68,60 +68,10 @@ A single unsigned integer seed value, different on each successive call
 pragma(inline, true)
 @property size_t unpredictableSeed() @trusted nothrow @nogc
 {
-    version(Windows)
-    {
-        import core.sys.windows.windows;
-        import core.sys.windows.winbase;
-        ulong ticks = void;
-        QueryPerformanceCounter(cast(long*)&ticks);
-    }
-    else
-    version(Darwin)
-    {
-        ulong ticks = mach_absolute_time();
-    }
-    else
-    version(Posix)
-    {
-        version(linux)
-            import core.sys.linux.time;
-        else
-        version(FreeBSD)
-            import core.sys.freebsd.time;
-        else
-        version(Solaris)
-            import core.sys.solaris.time;
-
-
-        import core.sys.posix.time;
-        timespec ts;
-        if(clock_gettime(CLOCK_MONOTONIC, &ts) != 0)
-        {
-            import core.internal.abort : abort;
-            abort("Call to clock_gettime failed.");
-        }
-        ulong ticks = (cast(ulong) ts.tv_sec << 32) ^ ts.tv_nsec;
-    }
-    version(Posix)
-    {
-        import core.sys.posix.unistd;
-        import core.sys.posix.pthread;
-        auto pid = cast(uint) getpid;
-        auto tid = cast(uint) pthread_self();
-    }
-    else
-    version(Windows)
-    {
-        auto pid = cast(uint) GetCurrentProcessId;
-        auto tid = cast(uint) GetCurrentThreadId;
-    }
-    ulong k = ((cast(ulong)pid << 32) ^ tid) + ticks;
-    k ^= k >> 33;
-    k *= 0xff51afd7ed558ccd;
-    k ^= k >> 33;
-    k *= 0xc4ceb9fe1a85ec53;
-    k ^= k >> 33;
-    return cast(size_t)k;
+    size_t seed;
+    //getRandomBlocking(&seed, seed.sizeof);
```

Comment:
1. Lets preserve old version if `getRandomBlocking` returns -1
2. and uncomment this line)